packages/core/src/shared/telemetry/vscodeTelemetry.json

```diff
@@ -275,6 +275,21 @@
             "name": "connectionState",
             "type": "string",
             "description": "A detailed state of a specific auth connection. Use `authStatus` for a higher level view of an extension's general connection."
+        },
+        {
+            "name": "callStack",
```

Comment:
How will this be used? Seems compatible (or should be compatible) with https://github.com/aws/aws-toolkit-vscode/pull/5442

Reply:
I've removed it for now but ideally we would be able to use it to help find the call path. IIRC Nik's changes require the functions to manually set their function name/class name so we can't use it for free